docs/pbc/all/carrier-management/third-party-integrations/paazl.md

```diff
@@ -32,7 +32,7 @@ YOUR ADVANTAGES:
 * Connectivity: Seamless integration with warehouse and customer service
 * Delivery Experience
 * Real-time Calculation: Show best delivery option based on order details
-* Arrival times: ETA’s based on carrier and custom business rules
+* Arrival times: ETAs based on carrier and custom business rules
```

Comment:
I thought the original (ETA's) meant plural, as there was no the in the start, and because shortening to 's is generally unadvised, to explain why.

If its singular ETA not ETA, then ETA is works perfectly though, so I will add 'the' and commit the suggestion!